jwql/website/apps/jwql/templates/base.html

```diff
@@ -178,20 +147,20 @@
 	    	</div>
 
 
-			<!--Show OAuth button when navbar is NOT collapsed-->
+			<!--Show GitHub button when navbar is NOT collapsed-->
 			<ul class="navbar-nav ml-auto d-none d-xl-block">
-				{% if user.ezid %}
-				<li class="nav-item active">
-					<a id="oauth_user" href="https://auth.mastdev.stsci.edu/tokens" target="_blank" rel="noopener noreferrer">
-						{{ user.ezid }}
+            	<li class="nav-item active d-none d-xl-block">
+					<a id="github-link"
+					   class="nav-link"
+					   href="https://github.com/spacetelescope/jwql"
+					   target="_blank">
+						<img src={{ static('img/githublogo.svg') }}
+							 height=35
+							 width=35
+							 alt="GitHub Logo"
+							 title="GitHub Logo">
```

Comment:
This bit is to just put the GitHub icon on the right-hand side of the navbar, where the login button used to be.

Reply:
It is nice to have the GitHub icon here!